#request
int32[] ids
bool images
bool scan
bool grid
bool user_data
---
#response
NodeData[] data
